jax/_src/pallas/triton/lowering.py

```diff
@@ -474,6 +474,65 @@ def _atomic_rmw(
       result_type, op, ptr, val, mask=mask, sem=semantic, scope=sync_scope
   )
 
+def _fp_bits_type(t: ir.Type) -> ir.Type:
```

Comment:
I think this can probably be `_cast(t, ir.IntegerType.get_signless(_element_type(t).width))`?